src/coreclr/src/vm/multicorejitplayer.cpp

```diff
@@ -965,6 +978,169 @@ DomainAssembly * MulticoreJitProfilePlayer::LoadAssembly(SString & assemblyName)
 }
 
 
+MethodDesc* MulticoreJitProfilePlayer::DecodeMethod(Module * pModule, SigPointer * pSigPointer)
```

Comment:
This is very similar to existing `ZapSig::DecodeMethod`. Would it make sense to add `ZapSig::Context` to the existing method to reuse the code?

Reply:
Actually it is very similar to Module::LoadIBCMethodHelper. I will try to reuse ZapSig::DecodeMethod or Module::LoadIBCMethodHelper. Thank you.